lib/preload.js

```diff
@@ -251,11 +265,19 @@ preload.CONTAINER_NAME = 'resin-image-preloader'
 
 /**
  * Build the preloader docker image
+ * @param {Object} docker - dockerode instance
+ * @param {Object} visuals - resin-cli-visuals (optional)
  * @returns {Stream}
  */
-preload.build = (docker) => {
+preload.build = (docker, visuals) => {
```

Comment:
There are 2 reasons:
1. the standalone mode (bin/resin-preload) doesn't use/need visuals;
2. To avoid having 2 copies of resin-cli-visuals if the versions diverge